pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/mapper/HadoopSegmentCreationMapReduceJob.java

```diff
@@ -273,12 +274,18 @@ protected String createSegment(String dataFilePath, Schema schema, Integer seqId
       return segmentName;
     }
 
-    private RecordReaderConfig getReaderConfig(FileFormat fileFormat) {
+    private RecordReaderConfig getReaderConfig(FileFormat fileFormat) throws IOException {
       RecordReaderConfig readerConfig = null;
       switch (fileFormat) {
         case CSV:
-          readerConfig = new CSVRecordReaderConfig();
-          break;
+          if(_readerConfigFile == null) {
+            readerConfig = new CSVRecordReaderConfig();
+	  }
+	  else {
+	    LOGGER.info("Reading CSV Record Reader Config from: {}", _readerConfigFile);
```

Comment:
is the readerconfig logged somewhere else? if not, I would log the deserialized readerConfig and readerConfigFile together.

Reply:
No it is not logged anywhere. I can log it in getReaderConfig